### PR TITLE
improved process_fasta.py

### PR DESCRIPTION
**Pull request by qiyunzhu**
Addressed issues #39 and #40 . @tkosciol @sjanssen2

### CODE REVIEW

**Comment by qiyunzhu** [comment]
@sjanssen2 thanks for flattering^^

File: microprot/scripts/process_fasta.py [hunk 1]
```diff
@@ -32,31 +33,77 @@ def extract_sequences(infile, identifiers=None):
     ----------
     infile : str
         file path to input multi-sequence FASTA file
-    identifiers : str or list of str (optional)
-        IDs (names) or indexes (n-th sequence in the file) of the proteins to
-        extract, may be:
-        1) Python list
-        2) comma-separated list
-        3) file path to the list (one ID or index per line)
+    identifiers :
```
Comment (tkosciol) [inline microprot/scripts/process_fasta.py]: that's super impressive! really comprehensive!

**Comment by tkosciol** [comment]
resolves issues #39 #40

**Comment by qiyunzhu** [comment]
@tkosciol My pleasure!

File: microprot/scripts/process_fasta.py (hunk 1 above)
Comment (qiyunzhu) [inline microprot/scripts/process_fasta.py]: thanks :)

**Pull request merged**